modules/timepoint_list/README.md

```diff
@@ -0,0 +1,48 @@
+# Timepoint List
+
+## Purpose
+
+The `timepoint_list` module serves as an entry-point for data entry
+about a candidate. It serves as a dashboard for a candidate linking
+to other modules that provide specific functionality about a candidate.
+
+## Intended Users
+
+The `timepoint_list` module is primarily intended for data entry
+staff who are accessing a candidate.
+
+## Scope
+
+The `timepoint_list` module provides a mechanism to link to
+information about candidates. 
+
+NOT in scope:
+
+The `timepoint_list` module does *not* provide any data entry
+functionality or mechanisms to configure study timepoints,
+only access timepoints of specific candidates.
```

Comment:
And Imaging data.  and  Candidate Info. 

Also should be mentioned, under scope or (technical details) interactions: 
* Basic candidate profile details are visible at top, including DoB. 
* Viewing at a glance the status of each timepoint -- including stage and status. 
* Behavioural Feedback 
* Bvl Exclusion -- > this should be mentioned under Interactions with Loris

Reply:
I don't see why people would think those things fall under the scope of this module. The timepoint related ones are mentioned because of the module name.